rollup.config.js

```diff
@@ -29,5 +30,8 @@ export default {
 
     // Resolve source maps to the original source
     sourceMaps(),
+
+    // Use absolute imports instead of relative imports
+    includePaths({ paths: ['./src'] }),
```

Comment:
Do we still need this even though we're using paths in tsc through Webpack?

Reply:
I was under the impression it was necessary, but after removing it and checking tests and storybook, it seems to work fine. I'll submit a commit removing it.